examples/metal-nimf-2-service-profile-connection/README.md

```diff
@@ -0,0 +1,13 @@
+# Metal to SERVICE_PROFILE Connection Example
```

Comment:
Service profile should be written like this: Generic Service Profile. Replace these words in the entire README.md file.